CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 8.5.0
+* Update dependency cryptography to v41.0.6 by @renovate in https://github.com/release-engineering/iib/pull/595
```

Comment:
I think a few other changes are missing from here

[Add guardrails for UniqueViolation caused by parallel requests](https://github.com/release-engineering/iib/commit/c6dd3fff0aaccb08d6f2cd7075d0f8cadf1c9897)

[Adding missing $ref for RegenerateBundleRequest](https://github.com/release-engineering/iib/commit/8414d96e1a1eb3529a8b2c9f69dae4e35932bb5f)

[Adding parameter ignore_bundle_ocp_version to merge-index API](https://github.com/release-engineering/iib/commit/bf50d7f103b91ab03bb4dbba616a0306c7cfa746)

Reply:
Thanks!

I'll add

Reply:
This commit [Adding missing $ref for RegenerateBundleRequest](https://github.com/release-engineering/iib/commit/8414d96e1a1eb3529a8b2c9f69dae4e35932bb5f) is still missing in the release notes :)